docs/logs.asciidoc

```diff
@@ -64,6 +65,14 @@ As a result, when an exception is reported to the logger:
 - Logger output will contain the `error.id` if the log format allows it (plaintext still requires some configuration)
 
 [float]
+[[log-sending]]
+=== Log sending (experimental)
+
+The agent can automatically capture and send logs directly to APM Server, which allows to ingest log events without relying filebeat.
+Log sending is controlled by the <<config-log-sending, `log_sending`>> configuration option and is disabled by default.
```

Comment:
Is it worth mentioning or linking the ECS reformatting options?

Reply:
Log sending is completely independent from ECS reformatting, so I think it's better to keep them apart to avoid confusion.